src/nvim/testdir/test_edit.vim

```diff
@@ -1534,3 +1534,27 @@ func Test_edit_noesckeys()
   bwipe!
   " set esckeys
 endfunc
+
+" Test for editing a directory
+func Test_edit_is_a_directory()
+  let dirname = getcwd() . "/Xdir"
```

Comment:
You removed `CheckEnglish` command. Port it in `check.vim`.

Reply:
I have ported it.